metron-interface/metron-rest-client/src/main/java/org/apache/metron/rest/model/pcap/PcapRequest.java

```diff
@@ -17,52 +17,57 @@
  */
 package org.apache.metron.rest.model.pcap;
 
-public class PcapRequest {
+// TODO reconcile with pcapmrjob
```

Comment:
Is this reconciliation part of this PR or part of the follow-on PR?

Reply:
Yes - handled here - https://github.com/apache/metron/pull/1109/files#diff-501443870e14e5528e28e1cf12f6a2a7